ratatui-core/src/buffer/buffer.rs

```diff
@@ -1249,27 +1247,7 @@ mod tests {
     // Prior to unicode-width 0.2, this was incorrectly detected as width 4 for some reason
     #[case::eye_speechbubble("👁️‍🗨️", "👁️‍🗨️xxxxx")]
     fn renders_emoji(#[case] input: &str, #[case] expected: &str) {
-        use unicode_width::UnicodeWidthChar;
-
-        dbg!(input);
-        dbg!(input.len());
-        dbg!(
-            input
-                .graphemes(true)
-                .map(|symbol| (symbol, symbol.escape_unicode().to_string(), symbol.width()))
-                .collect::<Vec<_>>()
-        );
-        dbg!(
-            input
-                .chars()
-                .map(|char| (
-                    char,
-                    char.escape_unicode().to_string(),
-                    char.width(),
-                    char.is_control()
-                ))
-                .collect::<Vec<_>>()
-        );
+        // Debug code removed - was causing compilation issues
```

Comment:
Please leave these `dbg!` in. These should be potentially fixed instead of removed.